backends-common/rabbitmq/pom.xml

```diff
@@ -93,6 +93,16 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-pool2</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.13</version>
```

Comment:
We already have that artifact with the same version declared in the root pom file, so you can omit the version here